### PR TITLE
Feature/update link

### PR DESCRIPTION
**Pull request by zuzannaszymanda**

### CODE REVIEW

**Review by anqaka — CHANGES_REQUESTED** [review]
secodary variant has wrong styles, we need styles like in alpaca theme for M2, please check and adjust

File: src/atoms/link/Link.vue [hunk 1]
```diff
@@ -10,4 +10,4 @@ export default {
 }
 </script>
 
-<style lang="scss" src="./Link.scss" />
+<style lang="css" src="./Link.css" />
```
Comment (szafran89) [inline src/atoms/link/Link.vue]: you can remove lang attr at all

Comment (szafran89) [reply src/atoms/link/Link.vue:13]: please remove, then merge to update branch

**Review by szafran89 — CHANGES_REQUESTED** [review]

File: src/atoms/link/Link.vue [hunk 2]
```diff
@@ -10,4 +10,4 @@ export default {
 }
```
Comment (szafran89) [inline src/atoms/link/Link.vue:9]: please remove empty line below `name: 'AlpacaLink',`

**Review by szafran89 — CHANGES_REQUESTED** [review]

**Comment by szafran89** [comment]
There is also issue related to UT, please check `yarn test`

**Review by szafran89 — APPROVED** [review]

**Comment by szafran89** [comment]
Looks like there is still issue related to UT, please `check yarn test`

**Comment by szafran89** [comment]
There is also no need to check the same thing in 2 tests
```
    expect(wrapper.is('a')).toBe(true)
    expect(wrapper.classes()).toContain('a-link')
```

**Review by szafran89 — APPROVED** [review]

**Comment by szafran89** [comment]
Ready to merge ;)

**Pull request merged**